docs/arlas-security.md

```diff
@@ -0,0 +1,90 @@
+# ARLAS security
+
+This page describes how to configure ARLAS in order to control access to resources.
+
+## Authentication
+Authentication is the process of identifying a user and is a prerequisite to [Authorisation](#authorisation).  
+Depending on the use case, some configuration must be set. Refer to [ARLAS server authentication configuration](arlas-server-configuration.md)
+
+* Use case 1: public access, no endpoint is protected (e.g. dev, test deployment)
+  - ARLAS server:  set `arlas_auth.enabled` to `false`
+  - ARLAS wui: do not set json property `arlas.authentication` in `config.json` (TODO: add link to WUI or add doc)
```

Comment:
@MohamedHamouGisaia do you confirm that's it's now in settings.yaml?

Reply:
Yes totally. I'll add a commit to fix that

Reply:
done